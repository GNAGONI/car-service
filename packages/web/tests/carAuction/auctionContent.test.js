import React from 'react'
import renderer from 'react-test-renderer'
import AuctionContent from '../../components/carAuction/auctionContent'

it('renders correctly', () => {
  const tree = renderer.create(<AuctionContent />).toJSON()
  expect(tree).toMatchSnapshot()
})
