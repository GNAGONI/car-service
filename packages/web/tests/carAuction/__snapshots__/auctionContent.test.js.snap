// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <section
        className="content auction-content pt-0 bg-before-default"
      >
        
        
      </section>
    </div>
  </div>
</div>
`;
